tests/integration/test_single_file_multiple_extensions.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+
+def test_multiple_file_extensions(testdir):
```

Comment:
I couldn't make your test fail on main, so I've replaced it with more of an integration test.

Reply:
I think you couldn't trigger the test because of a typo:
```python
class DotInFileExtension(SingleFileSnapshotExtension):
    _file_extension = "{file_extension}"
```
The `f` was missing before the f-string :) That was fixed when you moved it into an integration test setup.

Anyways, thank you for the merge!